aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -224,7 +225,7 @@ An exception is thrown if you attempt to configure the limit on a request after
 
 ### Minimum request body data rate
 
-[MinRequestBodyDataRate](/dotnet/api/microsoft.aspnetcore.server.kestrel.core.kestrelserverlimits.minrequestbodydatarate)  
+[MinRequestBodyDataRate](/dotnet/api/microsoft.aspnetcore.server.kestrel.core.kestrelserverlimits.minrequestbodydatarate)
```

Comment:
Leave the two spaces at the end ... it forces a `<br>` into the rendered topic.

Reply:
Ah I see, removing trailing spaces is a default in my editor.

Reply:
If you prefer, you can bullet them (and drop the spaces).

I'm sure I'll get a chance later to look at the whole topic and see if perhaps all of these cases should go to unordered lists. For now tho ... bare return is fine and matches what I did elsewhere.